web_monitoring/html_diff_render.py

```diff
@@ -102,7 +102,7 @@ def html_diff_render(a_text, b_text):
     replacements_new.update(replacements_old)
     soup_new = _add_undiffable_content(soup_new, replacements_new)
 
-    return soup_new.prettify(formatter='minimal')
+    return len(replacements_new), soup_new.prettify(formatter='minimal')
```

Comment:
This is not the right thing to count for this diff, but this diff is also undergoing massive changes, so we can fix it as part of those.

Reply:
Gotcha. I tried to figure this out but didn't think about it too hard. I should take the time to understand these internals better.

Reply:
:thumbsup: it’s a little complicated for sure. Hoping to add copious explanation and comments as I clean up #129.